src/custom/components/WalletModal/InjectedOption.tsx

```diff
@@ -23,10 +23,18 @@ const METAMASK_PROPS = {
   id: 'metamask',
 }
 
+const METAMASK_DEEP_LINK = 'https://metamask.app.link/dapp/'
+
 export function InstallMetaMaskOption() {
   return <Option {...METAMASK_PROPS} header={<Trans>Install MetaMask</Trans>} link={'https://metamask.io/'} />
 }
 
+export function OpenMetaMaskOption() {
```

Comment:
nit pick: Since this is an option  only expected in mobile. Consider renaming to `OpenMetaMaskMobileOption`